Source/WebCore/html/HTMLOptionElement.h

```diff
@@ -51,6 +51,8 @@ class HTMLOptionElement final : public HTMLElement {
 
     WEBCORE_EXPORT bool selected(AllowStyleInvalidation = AllowStyleInvalidation::Yes) const;
     WEBCORE_EXPORT void setSelected(bool);
+    WEBCORE_EXPORT bool selectedForBinding() const;
+    WEBCORE_EXPORT void setSelectedForBinding(bool);
```

Comment:
If I was doing this, I would probably have renamed the functions for internal use instead of renaming the ones for the bindings; requires a little bit of research about how many call sites those functions have. I’m not sure that "for binding" is the best way to distinguish these. Also, it’s only `setSelected` that needs a second version, and renaming the old `setSelected` would leave us with 3 functions rather than 4, which I think is better. Maybe `setSelectedWithoutSettingIsDirty` might be a good name, or we could find a shorter one. Really depends on how many call sites there are and why exactly it’s important to not set `m_isDirty` at those call sites.

I understand that the Chromium contributor chose this way of doing it.